### ISSUE TITLE
Controlnet latest update slow issue

### ISSUE THREAD

**Issue by nurullahozatak** [Mikubill/sd-webui-controlnet#1876]
With the last update of controlnet Stable Diffusion is getting slow after one image generation. Is there any solution ?
System: 3050ti, i711800h, 16GB ram, win10
There is just ControlNet extension
Latest WebUI and latest ControlNet 
I am using --xformers --medvram

Here is the export process (first and second generation)
First Generation:

2023-08-05 21:20:58,329 - ControlNet - INFO - Loading model: control_v11p_sd15_softedge [a8575a2a]
2023-08-05 21:21:00,006 - ControlNet - INFO - Loaded state_dict from [E:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_v11p_sd15_softedge.pth]
2023-08-05 21:21:00,007 - ControlNet - INFO - Loading config: E:\AI\stable-diffusion-webui\extensions\sd-webui-controlnet\models\control_v11p_sd15_softedge.yaml
2023-08-05 21:21:04,738 - ControlNet - INFO - ControlNet model control_v11p_sd15_softedge [a8575a2a] loaded.
2023-08-05 21:21:04,949 - ControlNet - INFO - Loading preprocessor: hed
2023-08-05 21:21:04,951 - ControlNet - INFO - preprocessor resolution = 544
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [00:16<00:00,  1.24it/s]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:20<00:00,  1.04s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [00:20<00:00,  1.55it/s]

Second Generation:

2023-08-05 21:23:09,090 - ControlNet - INFO - Loading model from cache: control_v11p_sd15_softedge [a8575a2a] 1.55it/s]
2023-08-05 21:23:09,092 - ControlNet - INFO - Loading preprocessor: hed
2023-08-05 21:23:09,094 - ControlNet - INFO - preprocessor resolution = 544
100%|██████████████████████████████████████████████████████████████████████████████████| 20/20 [03:25<00:00, 10.30s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [03:28<00:00, 10.41s/it]
Total progress: 100%|██████████████████████████████████████████████████████████████████| 20/20 [03:28<00:00, 10.35s/it]

**Comment by nurullahozatak** [comment]
Update: When I use --xformers --lowvram, it is working at the same speed as before, but it is noticeably slower compared to when using --medvram

**Comment by MrKuenning** [comment]
I am having similar results, first run took 5 seconds, second run took 3 minutes.
Its like there is a memory leak, because it doesnt release my vram, and tries to go way past my 10GB.

**Comment by harakiru** [comment]
Yeah there seems to be some kind of memory leak. If i enable controlnet and try to generate a 512x768 image with a fixed seed, the generation speed halves, and i cant use highres.fix. The problem is that the issue doesn't resolve after i disable controlnet, and the generation becomes non-deterministic. Only way to solve it seems to be shutting down webui.sh completely and enabling controlnet at all.
